multi-bam-quantify.wdl

```diff
@@ -4,17 +4,19 @@ import "tasks/biopet.wdl" as biopet
 import "tasks/htseq.wdl" as htseq
 
 workflow MultiBamExpressionQuantification {
-    Array[Pair[String,File]]+ bams
+    Array[Pair[String,Pair[File,File]]]+ bams #(sample, (bam, index))
```

Comment:
Can this be converted to a Map[String, File]?

Reply:
It would have to be a `Map[String, Pair[File, File]]`. We need both the bam and its index for each sample.
I haven't been able to find a way to produce a `Map` like this in the workflow which imports this one. In can generate an array of pairs, like the one used as input now. @rhpvorderman suggested using a python script to write the array of pairs to a json and reading that json into a map using `read_json()`. But this won't load it in as an `Map[String, Pair[File, File]]`, but as an `Object`.

Reply:
For now we keep it like this, we need to look into structs anyway at some point